docs/source/conventions.rst

```diff
@@ -6,25 +6,36 @@ Git branching and merging
 
 We adopt the following system for branching and merging:
 
-1. *master* branch: numbered releases of the code. Never edited. Merged from *develop* and *bug fix branches*. Long living.
-2. *develop* branch: rather stable version of code under development. Never edited. Merged from topic specific issue branches. Long living.
-3. *issuexx-short-heading* - issue specific branches. Main working area. Short living. Merged into develop.
-4. Never edit code in the master or develop branch. Always make a new branch for your edits.
-5. New branch should be very specific to only one problem. It should be really short living.
-6. Commit often.
-7. Branch often. Merge after completion of the task for this branch, then delete the branch.
-8. Branch only from master or from develop.
-9. HOTFIX workflow
-   a) branch from master into hotfixNNN_title (NNN = ticket number);
-   b) update tests, fix the bug, increment micro version. Commit to hotfixNNN_title,
-      push and verify that Travis passes tests;
-   c) merge master into your branch (if there have been commits in master since you branched),
-      push and verify that Travis passes tests;
-   d) merge into master with --ff-only, push and verify that Travis passes tests;
-   e) merge master into develop, push and verify that Travis passes tests;
-   f) delete the hotfix branch;
-
-
+1. **master** branch: numbered releases of the code. Never edited. Merged from *develop* and *bug fix* or *hot fix* branches (see notes on workflow below). Long living.
+2. **develop** branch: rather stable version of code under development. Never edited. Merged from topic specific issue branches. Long living.
+3. **issueNNN-short-heading** - issue specific branches (NNN = issue number). Main working area. Short living. Branched from, and merged back into develop.
+4. **hotfixNNN_short-heading** - branches that are specific to an hotfix issue. Hotfixes are bugfixes on master that can not wait until next release. (should be branched from master and merged back ASAP)
```

Comment:
Should be written in the same way as the others. i.e. remove the parenthesis and rewrite slightly.
These should also then be merged into develop by merging master into develop. Although I would absolutely prefer a rebase solution.

Reply:
done in next commit